ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/ccache/tasks/main.yml

```diff
@@ -11,30 +11,40 @@
 - name: Download ccache.tar.gz
   get_url:
     url: https://www.samba.org/ftp/ccache/ccache-3.4.2.tar.gz
-    dest: /home/{{ Jenkins_Username }}/ccache.tar.gz
+    dest: /tmp/ccache.tar.gz
     mode: 0440
     validate_certs: False
   when: ccache_status.stat.isdir is not defined
```

Comment:
Missed the check:

```
- name: Check if ccache is already installed
  stat:
path: /home/{{ Jenkins_Username }}/ccache-3.4.2
```

You might want to check if ccache exists in `/usr/local/bin/ccache`

Reply:
Updated.